backend/app.js

```diff
@@ -1,6 +1,8 @@
 import express from 'express';
 import cors from 'cors';
 
+const prisma = require('./db/db');
```

Comment:
lets try to stay consistent with how we import modules. Right here, you're using the commonJS module system which uses the `require()` syntax, but on line 1 and 2, you're using the ES Module to import modules with `import`. i would recommend using `import` instead of `require()` because it's a newer and more standardized system. 

in this case, you can do:

```js
import prisma from './db'
```